app/src/main/java/com/example/tour_a_versity_v3/MapsActivity.kt

```diff
@@ -11,7 +11,10 @@ import com.google.android.gms.maps.model.LatLng
 import com.google.android.gms.maps.model.Marker
 import com.google.android.gms.maps.model.MarkerOptions
 
-class MapsActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnInfoWindowClickListener {
+//class MapsActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnInfoWindowClickListener {
+class MapsActivity :
```

Comment:
Separate the class header and body when the class header is long. Put a blank line following the class header as described here: https://kotlinlang.org/docs/reference/coding-conventions.html

Reply:
Good idea, this makes it more readable.